tests/devapps/UWP/MainPage.xaml

```diff
@@ -11,6 +11,8 @@
     <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
         <Button Content="Acquire Token Interactive" HorizontalAlignment="Center" Margin="0,100,0,0" VerticalAlignment="Top" Click="AccessTokenButton_ClickAsync"/>
         <Button Content="Acquire Token Silent" HorizontalAlignment="Center" Margin="0,150,0,0" VerticalAlignment="Top" Click="AccessTokenSilentButton_ClickAsync"/>
+        <Button Content="Acquire Token with ADAL" HorizontalAlignment="Center" Margin="0,257,0,0" VerticalAlignment="Top" Click="ADALButton_ClickAsync"/>
```

Comment:
The rest of the changes are that I modified the UWP sample app to use both ADAL and MSAL to test legacy token cache

Reply:
awesome

---
In reply to: [250237656](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/824#discussion_r250237656) [](ancestors = 250237656)